apisix/cli/env.lua

```diff
@@ -89,6 +90,12 @@ return function (apisix_home, pkg_cpath_org, pkg_path_org)
     local openresty_args = openresty_path_abs .. [[ -p ]] .. apisix_home .. [[ -c ]]
                            .. apisix_home .. [[/conf/nginx.conf]]
 
+    local apisix_daemon = true
```

Comment:
As I mentioned many times in
https://github.com/apache/apisix/pull/8681#issuecomment-1384864654
https://github.com/apache/apisix/pull/8681#discussion_r1073065444

Could we add a custom configuration to let APISIX don't run as a daemon, just in the test?